demo/Demo/Textfields.elm

```diff
@@ -224,6 +225,32 @@ textfields model =
         ]
        """
     )
+  , ( "Expandable textfield"
+    , Textfield.render Mdl [11] model.mdl
+        [ Textfield.label "Expandable"
+        , Textfield.floatingLabel
+        -- NOTE: The id's must match
```

Comment:
Expandable actually sets the id but if users for some reason provide `Options.inner` with ID then those ids must match